homeassistant/components/alphaess/coordinator.py

```diff
@@ -0,0 +1,48 @@
+"""Coordinator for AlphaEss integration."""
+
+from datetime import timedelta
+import logging
+
+from alphaess import alphaess
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator, UpdateFailed
+
+from .const import DOMAIN
+
+_LOGGER: logging.Logger = logging.getLogger(__package__)
+SCAN_INTERVAL = timedelta(minutes=5)
+PLATFORMS = [Platform.SENSOR]
```

Comment:
Move those parts to const or `__init__`.

Reply:
I have moved them to const